worker/uniter/remotestate/snapshot.go

```diff
@@ -93,6 +93,9 @@ type Snapshot struct {
 	// UpgradeSeriesStatus is the preparation status of any currently running
 	// series upgrade
 	UpgradeSeriesStatus model.UpgradeSeriesStatus
+
+	// RunningStatus is set on CAAS models for remote init/upgrade of charm.
+	RunningStatus *RunningStatus
```

Comment:
Maybe ContainerRunningStatus to make it clearer? Or PodRunningStatus?
It gives the attr name more context